### PR TITLE
Feat/typography component

### PR DESCRIPTION
**Pull request by mczapkowicz**
Part of chart theming: https://metakeen.atlassian.net/jira/software/projects/FEE/boards/10?selectedIssue=FEE-645

### CODE REVIEW

File: packages/ui-core/src/components/typography-settings/typography-settings.component.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  colorSuggestions?: string[];
+  /** Font size suggestions that will be available in font size dropdown */
+  fontSizeSuggestions?: number[];
+  /** Ref to scrollable parent element - it can be used to hide color picker or dropdowns on scroll event */
+  scrollableContainerRef?: RefObject<HTMLDivElement>;
+};
+
+const TypographySettings: FC<Props> = ({
+  settings,
+  onChange,
```
Comment (maciejrybaniec) [inline packages/ui-core/src/components/typography-settings/typography-settings.component.tsx:25]: As we discussed - could You please add possibility to specify which properties are allowed to edit like for eg. text alignment etc.

**Review by maciejrybaniec — CHANGES_REQUESTED** [review]

File: packages/ui-core/src/components/typography-settings/components/attribute-dropdown/attribute-dropdown.styles.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+import { colors } from '@keen.io/colors';
+
+export const Container = styled.div`
+  color: ${colors.black[100]};
+  font-size: 14px;
+  font-family: 'Lato Regular', sans-serif;
```
Comment (apryka) [inline packages/ui-core/src/components/typography-settings/components/attribute-dropdown/attribute-dropdown.styles.ts:7]: please use `BodyText` from `keen.io/typography` here

File: packages/ui-core/src/components/typography-settings/components/font-size/font-size.styles.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { colors } from '@keen.io/colors';
+
+export const Header = styled.div`
+  display: flex;
+  align-items: center;
+
+  font-size: 14px;
```
Comment (apryka) [inline packages/ui-core/src/components/typography-settings/components/font-size/font-size.styles.ts:8]: keen.io/typography

File: packages/ui-core/src/components/typography-settings/components/option/option.component.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import styled, { css } from 'styled-components';
+import { transparentize } from 'polished';
+import { colors } from '@keen.io/colors';
+
+const Option = styled.div<{ isActive?: boolean }>`
+  padding: 5px 14px;
+  font-size: 14px;
+  font-family: 'Lato Regular', sans-serif;
```
Comment (apryka) [inline packages/ui-core/src/components/typography-settings/components/option/option.component.ts:8]: keen.io/typography

**Review by apryka — APPROVED** [review]

**Review by maciejrybaniec — APPROVED** [review]

**Pull request merged**